mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return trimWord();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  return a + b + c;
```

Comment:
This is syntactically correct, just instead of pluses it should have been multiplication